### ISSUE TITLE
Bug in the Italian translation file woocommerce-it_IT.po

### ISSUE THREAD

**Issue by astronim** [woothemes/woocommerce#37256]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The woocommerce-it_IT.po translation file contains the following fragment;

`#. translators: %s: regen thumbs url
#: includes/customizer/class-wc-shop-customizer.php:535
msgid "After publishing your changes, new image sizes may not be shown until you <a href=\"%s\" target=\"_blank\">Regenerate Thumbnails</a>."
msgstr "Dopo aver pubblicato le modifiche, le nuove dimensioni dell'immagine non saranno utilizzate finché non avrai utilizzato <a href=\"%2$s\" target=\"_blank\">Rigenera miniature</a>."
`
You need to replace **%2$s** with **%s** otherwise a fatal error occurs on the site under certain conditions.

### Expected behavior

Correct translation into Italian

### Actual behavior

The translation file contains an error

### Steps to reproduce

1. Activate Storefront theme.
2. Turn on the Italian language on the site. The backend language should also be Italian.
3. Download all Translations in Dashboard->Update. 
4. Add the following code to the child or main theme `add_filter( 'woocommerce_background_image_regeneration', '__return_false' );`
5. Disable the Jetpack plugin if you have one.
6. Go to Appearance -> Customizer and you will see this fatal error: `Fatal error: Uncaught ArgumentCountError: 3 arguments are required, 2 given in  /wp-content/plugins/woocommerce/includes/customizer/class-wc-shop-customizer.php:535`

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): http://localhost/test
Site address (URL): http://localhost/test
WC Version: 7.5.0
REST API Version: ✔ 7.5.0
WC Blocks Version: ✔ 9.6.5
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.1.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: –
Language: it_IT
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.46 (Unix) mod_fastcgi/mod_fastcgi-SNAP-0910052141 PHP/8.0.8 OpenSSL/1.0.2u mod_wsgi/3.5 Python/2.7.18
PHP Version: 8.0.8
PHP Post Max Size: 512 MB
PHP Time Limit: 60
PHP Max Input Vars: 10000
cURL Version: 7.68.0
OpenSSL/1.0.2u

SUHOSIN Installed: –
MySQL Version: 5.7.34
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.5.0
WC Database Prefix: wp_demo1_
Dimensione totale database: 20.10MB
Dimensione dati database: 18.16MB
Dimensione indice database: 1.94MB
wp_demo1_woocommerce_sessions: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_woocommerce_api_keys: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_woocommerce_attribute_taxonomies: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_woocommerce_downloadable_product_permissions: Dati: 0.02MB + indice: 0.06MB + motore InnoDB
wp_demo1_woocommerce_order_items: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_woocommerce_order_itemmeta: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_woocommerce_tax_rates: Dati: 0.02MB + indice: 0.06MB + motore InnoDB
wp_demo1_woocommerce_tax_rate_locations: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_woocommerce_shipping_zones: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_woocommerce_shipping_zone_locations: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_woocommerce_shipping_zone_methods: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_woocommerce_payment_tokens: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_woocommerce_payment_tokenmeta: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_woocommerce_log: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_actionscheduler_actions: Dati: 0.02MB + indice: 0.13MB + motore InnoDB
wp_demo1_actionscheduler_claims: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_actionscheduler_groups: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_actionscheduler_logs: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_commentmeta: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_comments: Dati: 0.02MB + indice: 0.09MB + motore InnoDB
wp_demo1_e_events: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_links: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_options: Dati: 4.06MB + indice: 0.06MB + motore InnoDB
wp_demo1_postmeta: Dati: 6.31MB + indice: 0.30MB + motore InnoDB
wp_demo1_posts: Dati: 6.50MB + indice: 0.06MB + motore InnoDB
wp_demo1_revslider_css: Dati: 0.13MB + indice: 0.02MB + motore InnoDB
wp_demo1_revslider_css_bkp: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_revslider_layer_animations: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_revslider_layer_animations_bkp: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_revslider_navigations: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_revslider_navigations_bkp: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_revslider_sliders: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_revslider_sliders_bkp: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_revslider_slides: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_revslider_slides_bkp: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_revslider_static_slides: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_revslider_static_slides_bkp: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_termmeta: Dati: 0.05MB + indice: 0.03MB + motore InnoDB
wp_demo1_terms: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_term_relationships: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_term_taxonomy: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_usermeta: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_users: Dati: 0.02MB + indice: 0.05MB + motore InnoDB
wp_demo1_wc_admin_notes: Dati: 0.05MB + indice: 0.00MB + motore InnoDB
wp_demo1_wc_admin_note_actions: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_wc_category_lookup: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_wc_customer_lookup: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_wc_download_log: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_wc_order_coupon_lookup: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_wc_order_product_lookup: Dati: 0.02MB + indice: 0.06MB + motore InnoDB
wp_demo1_wc_order_stats: Dati: 0.02MB + indice: 0.05MB + motore InnoDB
wp_demo1_wc_order_tax_lookup: Dati: 0.02MB + indice: 0.03MB + motore InnoDB
wp_demo1_wc_product_attributes_lookup: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_wc_product_download_directories: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_wc_product_meta_lookup: Dati: 0.02MB + indice: 0.09MB + motore InnoDB
wp_demo1_wc_rate_limits: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_wc_reserved_stock: Dati: 0.02MB + indice: 0.00MB + motore InnoDB
wp_demo1_wc_tax_rate_classes: Dati: 0.02MB + indice: 0.02MB + motore InnoDB
wp_demo1_wc_webhooks: Dati: 0.02MB + indice: 0.02MB + motore InnoDB

### Post Type Counts ###

attachment: 289
custom_css: 1
elementor_library: 2
html_block: 10
nav_menu_item: 49
page: 28
post: 9
product: 31
product_variation: 3
shop_coupon: 1
wp_global_styles: 1
wpcf7_contact_form: 3

### Security ###

Secure connection (HTTPS): ❌
					Il tuo negozio non utilizza HTTPS. Scopri di più su HTTPS e certificati SSL .
Hide errors from visitors: ❌I messaggi di errore non saranno mostrati ai visitatori.

### Active Plugins (1) ###

WooCommerce: by Automattic – 7.5.0

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 0
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: –
Order datastore: WC_Order_Data_Store_CPT

### WC Pages ###

Shop base: #3121 - /shop/
Carrello: #3122 - /cart/
Pagamento: #3123 - /checkout/
Il mio account: #3124 - /my-account/
Termini e condizioni: #3107 - /privacy-policy/

### Theme ###

Name: Storefront
Version: 4.2.0
Author URL: https://woocommerce.com/
Child Theme: ❌ – Se stai modificando WooCommerce o un tema genitore che non hai costruito personalmente
ti consigliamo di utilizzare un tema child. Vedi: Come creare un tema child

WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
multichannel-marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
new-product-management-experience
product-variation-management
settings

Daily Cron: ✔ Next scheduled: 2023-03-16 06:56:12 +00:00
Options: ✔
Notes: 50
Onboarding: completed

### Action Scheduler ###

Completato: 36
Oldest: 2023-02-13 04:32:42 +0000
Newest: 2023-03-16 01:57:57 +0000

Operazione non riuscita: 1
Oldest: 2023-03-16 01:59:55 +0000
Newest: 2023-03-16 01:59:55 +0000

In attesa: 1
Oldest: 2023-03-16 08:19:56 +0000
Newest: 2023-03-16 08:19:56 +0000

### Status report information ###

Generated at: 2023-03-16 02:00:39 +00:00
`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by DAnn2012** [comment]
Hello @astronim 

I have submitted the change to translate.wordpress.org

https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/it/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=5702322&filters%5Btranslation_id%5D=102752403

https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/it/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=5702271&filters%5Btranslation_id%5D=102752412

I hope it will be approved soon by the WordPress Polyglots Team.

I thank you for the report.

**Comment by DAnn2012** [comment]
Hello @astronim 

The string was approved in Stable (latest release): Italian

https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/it/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=5702322&filters%5Btranslation_id%5D=102752403

**Comment by coreymckrill** [comment]
This appears to be fixed now, but feel free to reopen if there's still an issue.
